src/reports/client/main.cpp

```diff
@@ -0,0 +1,198 @@
+#include <iostream>
```

Comment:
applications should probably live somewhere higher up in the source tree

Reply:
I agree, but on the other hand this is only related to reports...thats why i kept it there
Maybe a folder called apps on src? (src/apps/reports/main.cpp)
I dont know what would be the best to be honest.

Reply:
Isn't this sort of a supporting tool? Then `libsonata/utils` or so seems more appropriate.